zk_toolbox/crates/zk_supervisor/src/commands/test/args/recovery.rs

```diff
@@ -1,12 +1,14 @@
 use clap::Parser;
 use serde::{Deserialize, Serialize};
 
-use crate::messages::{MSG_NO_DEPS_HELP, MSG_TESTS_RECOVERY_SNAPSHOT_HELP};
+use crate::messages::{MSG_NO_DEPS_HELP, MSG_NO_KILL_HELP, MSG_TESTS_RECOVERY_SNAPSHOT_HELP};
 
 #[derive(Debug, Serialize, Deserialize, Parser)]
 pub struct RecoveryArgs {
     #[clap(short, long, help = MSG_TESTS_RECOVERY_SNAPSHOT_HELP)]
     pub snapshot: bool,
     #[clap(short, long, help = MSG_NO_DEPS_HELP)]
     pub no_deps: bool,
+    #[clap(short, long, help = MSG_NO_KILL_HELP)]
+    pub auto_kill: bool,
```

Comment:
no kill

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2731/commits/1d5c81092c3afb4a58541f696356855d07f7a1c6